tartufo/config.py

```diff
@@ -278,12 +278,15 @@ def compile_rules(patterns: Iterable[Dict[str, str]]) -> List[Rule]:
             raise ConfigException(
                 f"Invalid value for match-type: {pattern.get('match-type')}"
             ) from exc
-        try:
-            scope = Scope(pattern.get("scope", Scope.Line.value))
-        except ValueError as exc:
-            raise ConfigException(
-                f"Invalid value for scope: {pattern.get('scope')}"
-            ) from exc
+        if exclude_type != "regex":
+            try:
+                scope = Scope(pattern.get("scope", Scope.Line.value))
+            except ValueError as exc:
+                raise ConfigException(
+                    f"Invalid value for scope: {pattern.get('scope')}"
+                ) from exc
+        else:
+            scope = Scope.Line
```

Comment:
This would be more direct this way:
```suggestion
        if exclude_type == "regex":
            # regex exclusions always have line scope
            scope = Scope.Line
        else:
            # entropy exclusions can specify scope
            try:
                scope = Scope(pattern.get("scope", Scope.Line.value))
            except ValueError as exc:
                raise ConfigException(
                    f"Invalid value for scope: {pattern.get('scope')}"
                ) from exc
```

Reply:
good catch, agree here. ended up focusing on the unit tests that i didn't self-scrutinize the code changes as much as i should have